cranelift/isle/docs/language-reference.md

```diff
@@ -262,19 +262,19 @@ subterms:
 * `_` is a wildcard and matches anything, without capturing it.
 
 * Primitive constant values, such as `42` or `$Symbol`, match only if
-  the term is exactly equal to this constant.
+  the term is an exact equivalent.
```

Comment:
"an exact equivalent" does not read the same as "is exactly equal" to me: the former implies something about an equivalence class, different forms with the same underlying denotation, etc., whereas for primitive values (e.g. integers) we really do want to say "equal". I'm not sure what "equivalent" does better here.